.rubocop.yml

```diff
@@ -146,6 +146,11 @@ Style/HashSyntax:
 # This is incompatible with our other migration checks
 Rails/BulkChangeTable:
   Enabled: false
+
+# Incompatible with DfE-Analytics custom matcher we have extended (have_been_enqueued_as_analytics_event)
```

Comment:
could we mark these as exceptions instead of having this in the global config?

Reply:
I was hesitant to do this as they're a lot of them, but lets give it a try :eyes: